### DISCUSSION TITLE
Programação

### DISCUSSION THREAD

**Discussion by JVzin0070** [community/community#143022] [Programming Help]
### Body

Olá pessoal!
Preciso de ajuda com programação. Estou tentando fazer um programa tipo o mindmeister para criação de mapas mentais.
No caso eu quero rodar ele localmente pois será para uso pessoal, então não vai precisar de todas as ferramentas..
Eu sei muito pouco sobre programação, então resolvi pedir ajuda aqui!
Agradeço imensamente se alguém aceitar esse desafio!.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghostinhershell** [comment]
Hi @JVzin0070, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
